backend/sozisel/lib/sozisel_web/schema/types/whiteboard_types.ex

```diff
@@ -0,0 +1,38 @@
+defmodule SoziselWeb.Schema.Types.WhiteboardTypes do
+  use SoziselWeb.Schema.Notation
+
+  object :whiteboard do
+    field :task, non_null(:string)
+  end
+
+  input_object :whiteboard_input do
+    field :task, non_null(:string)
+  end
+
+  input_object :create_whiteboard_input do
+    import_fields :create_event_input_base
+
+    field :event_data, non_null(:whiteboard_input)
+  end
+
+  input_object :update_whiteboard_input do
+    import_fields :update_event_input_base
+
+    field :event_data, non_null(:whiteboard_input)
+  end
+
+  object :whiteboard_result do
+    field :path, non_null(:string)
```

Comment:
I know that I forgot it with recording path but please add `@desc` telling that it is relative path under which the media object should be accessed.